### DISCUSSION TITLE
Great Work

### DISCUSSION THREAD

**Discussion by mickdelaney** [scottiemc7/hotwire-aspnet-demo-chat#1] [General]
This is really quite cool. Great Work !!
Its the first time I've seen a hotwire attempt on asp.net core.
any plans to take it further ?
what would you do as a next step, i might be interested in collaborating !!!

**Comment by scottiemc7** [comment]
Hi, thank you for your kind words. I couldn't find any other attempts to port Hotwire over to the Microsoft stack. Hotwire seems like an amazing project and I look forward to using it for real at some point in the future. I don't plan to put much more work into this demo, but I welcome pull requests if you have any ideas on how to move it forward. The front end is a bit of a mess and I am annoyed to have to rely on something as heavy handed as [Fluid](https://github.com/sebastienros/fluid) for Turbo Streams. One of those tasks could be a good starting point.
